src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs

```diff
@@ -48,6 +48,8 @@ public Startup(IOptions<AdminOptions> adminOptions)
 
         public override void ConfigureServices(IServiceCollection services)
         {
+            services.AddTransient<IConfigureOptions<CulturePickerOptions>, CulturePickerOptionsConfiguration>();
```

Comment:
just use `services.Configure<CulturePickerOptions>`

Reply:
Yes, if I remove the above class